plugins/splunk-on-call/src/plugin.ts

```diff
@@ -49,3 +50,14 @@ export const SplunkOnCallPage = splunkOnCallPlugin.provide(
     mountPoint: rootRouteRef,
   }),
 );
+
+export const SplunkOnCallCard = splunkOnCallPlugin.provide(
```

Comment:
These are commonly prefixed with the word `Entity` to signify that it's meant to be used in the catalog entity view.

Reply:
Right, renamed the actual file but didn't update the export here. Fixed 👍